README.md

```diff
@@ -130,7 +130,26 @@ Response time metrics are based on the **metrics** variable, and the values can
 
 The board is structured into 4 sections
 
-#### Apex Overview
+#### Performance Overview
+
+[<img src="./images/prometheus-dashboard-k6-test-result-performance.png" width="500"/>](./images/prometheus-dashboard-k6-test-result-performance.png)
+
+#### HTTP
+
+[<img src="./images/prometheus-dashboard-k6-test-result-http.png" width="500"/>](./images/prometheus-dashboard-k6-test-result-http.png)
+
+#### Scenarios
+
+[<img src="./images/prometheus-dashboard-k6-test-result-scenarios.png" width="500"/>](./images/prometheus-dashboard-k6-test-result-scenarios.png)
+
+### Test list dashboard
+
+[<img src="./images/prometheus-dashboard-k6-test-runs.png" width="500"/>](./images/prometheus-dashboard-k6-test-runs.png)
```

Comment:
I don't think we need these details in the README. The performance overview should be enough as the screenshot for the _Test result dashboard_.

Reply:
ok